src/styles.css

```diff
@@ -0,0 +1,200 @@
+@import url(https://fonts.bunny.net/css?family=open-sans:400,800);
```

Comment:
Is there a strong reason to use an external font file ?

Reply:
No, just convenience. I did pick bunny over Google fonts for GDPR reasons but did not want to look into font licenses when I wrote that bit of code.

Reply:
Fanciness! Seriously though, it would have felt strange to not set a font when the goal is to achieve a more professional look. But I'm open to your suggestions :)

If you'd like to stick with Open Sans, I believe we could self-host it after downloading WOFF2 files from here: https://gwfh.mranftl.com/fonts/open-sans?subsets=latin
Here's the license: https://github.com/googlefonts/opensans/blob/main/OFL.txt